core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -56,6 +57,12 @@
                                                                    DEFAULT_DEFAULT_BACKOFF_MAX_ATTEMPTS,
                                                                    value -> value > 0);
 
+    private static final int DEFAULT_DEFAULT_EXPONENTIAL_BACKOFF_START_INTERVAL = 10;
+    private static final int DEFAULT_EXPONENTIAL_BACKOFF_START_INTERVAL =
+            getInt("defaultExponentialBackoffStartInterval",
+                   DEFAULT_DEFAULT_EXPONENTIAL_BACKOFF_START_INTERVAL, value -> value > 0);
```

Comment:
- How about: `defaultExponentialBackoffInitialDelayMillis` ?
- The initial delay of 10 milliseconds seems too short as a default value. How about 1000ms?
- `value >= 0` (0 is allowed in `ExponentialBackoff`)
- How about adding the property for the multiplier of `ExponentialBackoff`?